finalPrep/README.md

```diff
@@ -31,24 +31,26 @@ In Mod 0 you've learned about different techniques for managing your time at Tur
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+My calendar is a bit incomplete and starts at the wrong cohort (I deferred to 2111), but that was the only calendar available through the link provided.
+- ![Screen Shot 2021-09-26 at 1 10 35 AM](https://user-images.githubusercontent.com/90285348/134799191-a47d7f1a-a489-4eef-bf38-f5308b6ec36b.png)
```

Comment:
I am glad you've synced with the mod 1 calendar, but I suggest fleshing out your personal schedule a bit. When do you plan on studying? There will be a lot of time that needs to be put in outside of class.